app/views/layouts/application.html.erb

```diff
@@ -6,59 +6,72 @@
 	<%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
 	<%= csrf_meta_tags %>
 </head>
-<body>
+<body onload="getTimer();">
```

Comment:
Das ist nicht so schön. Besser sowas wie

```javascript
$(document).ready(function () {
  getTimer();
});
```
in die `application.js` schreiben.